### PR TITLE
Add core typography tokens for iOS

### PR DESCRIPTION
**Pull request by janogarcia**
Core typograhpy tokens for iOS. Most useful for generating Figma Styles.

Please note that it doesn't include styles (composite tokens) for monospace fonts and uppercase text. Both are planned, though.

### CODE REVIEW

File: assets/ios/swift/CpdDark.swift [hunk 1]
```diff
@@ -160,6 +160,35 @@ public class CpdDarkDesignTokens {
...
+    public static let cpdFontLineHeightHeadingMdRegular = 28
+    public static let cpdFontLineHeightHeadingSmRegular = 25
+    public static let cpdFontLineHeightBodyLgRegular = 22
+    public static let cpdFontLineHeightBodyMdRegular = 20
+    public static let cpdFontLineHeightBodySmRegular = 18
+    public static let cpdFontLineHeightBodyXsRegular = 16
+    public static let cpdFontWeightBold = 700
+    public static let cpdFontWeightSemibold = 600
+    public static let cpdFontWeightRegular = 400
+    public static let cpdFontFamilyMono = SF Mono
```
Comment (gsouquet) [inline assets/ios/swift/CpdDark.swift:190]: Does not look like this is going to compile

Comment (janogarcia) [reply assets/ios/swift/CpdDark.swift:190]: @gsouquet We basically need to define how do we want to provide tokens to iOS platform. I don't think atomic typography tokens like letter spacing, font size, and such are going to be useful for their needs (that's why I added _"Most useful for generating Figma Styles."_, as it's the immediate priority).

Maybe the just need to know how our composite tokens map to built-in _text styles_, and maybe even when to combine them with system-provided modifiers (aka _symbolic traits_) like bold or tight leading (if we add support for it).

https://developer.apple.com/design/human-interface-guidelines/foundations/typography/#using-system-fonts

Comment (janogarcia) [reply assets/ios/swift/CpdDark.swift:190]: @gsouquet Actually, that's also why I initially labelled atomic typography tokens for iOS as N/A in the [internal doc on Typography tokens](https://github.com/vector-im/compound-internal/blob/internal-docs/docs/typography.md#ios) transformations.

Comment (gsouquet) [reply assets/ios/swift/CpdDark.swift:190]: sorry that was a very unhelpful comment from my side. It was more a note to myself to review this.

I've schedule a call with doug tomorrow to go through all of that

Comment (janogarcia) [reply assets/ios/swift/CpdDark.swift:190]: Awesome. 👍

Comment (janogarcia) [reply assets/ios/swift/CpdDark.swift:190]: I'd love to join the call, if possible, but it's going to be difficult tomorrow (already quite a lot of calls scheduled), so please let me know of any outcomes/questions afterward.

**Review by gsouquet — APPROVED** [review]

**Comment by gsouquet** [comment]
LGTM from the token side of thing, the generated assets will need to change, but no need to block this PR on that.

**Pull request merged**